### PR TITLE
misc bindings to podman v3

### PR DESCRIPTION
**Pull request by baude**
manifest, system, info, volumes, play,  and generate bindings are
updated to always have binding options.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Why are we including generated code i the repo?

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

File: pkg/bindings/generate/generate.go [hunk 1]
```diff
@@ -4,31 +4,23 @@ import (
...
 	"net/http"
-	"net/url"
-	"strconv"
 
 	"github.com/containers/podman/v2/pkg/bindings"
 	"github.com/containers/podman/v2/pkg/domain/entities"
 )
 
-func Systemd(ctx context.Context, nameOrID string, options entities.GenerateSystemdOptions) (*entities.GenerateSystemdReport, error) {
+func Systemd(ctx context.Context, nameOrID string, options *SystemdOptions) (*entities.GenerateSystemdReport, error) {
```
Comment (vrothberg) [inline pkg/bindings/generate/generate.go:12]: `entities.GenerateSystemdReport` as a return value is escaping the bindings.  Are there plans to do the same dance for return values?

Comment (baude) [reply pkg/bindings/generate/generate.go:12]: i cannot see this getting done in this round.

**Pull request merged**